lib/ssl/src/ssl_crl_cache_api.erl

```diff
@@ -70,10 +75,12 @@ omitting `Issuer`. For compatibility, this is still supported: if there is no
 It is possible to return logger info that will be used by the TLS connection to
 produce log events.
 """.
--doc(#{since => <<"OTP 18.0,OTP 19.0,OTP 22.2">>}).
--callback lookup(dist_point(), issuer_name(), crl_cache_ref()) ->
+-callback lookup(Distpoint::dist_point(), Issuer::public_key:issuer_name(), CacheRef::crl_cache_ref()) ->
     not_available | [public_key:der_encoded()] |
     {{logger, logger_info()}, [public_key:der_encoded()]}.
+
+
+-doc(#{since => <<"OTP 18.0,OTP 22.2">>}).
```

Comment:
```suggestion
-doc(#{since => ~"OTP 18.0"}).
```

There should only be one since version, the `OTP 22.2` should be mentioned in text when that specific return was added. There are many places where this mistake is done, I've only commented on this one.

Reply:
fixed